packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -403,7 +397,9 @@ export class DataProxyEngine extends Engine {
           }
         }
 
-        this.logEmitter.emit('warn', { message: 'This request can be retried' })
+        this.logEmitter.emit('warn', {
```

Comment:
I don't think sentence was documented but let's check if that was the case

Reply:
Good call: [It was not documented](https://www.google.com/search?q=%22can+be+retried%22+site%3Aprisma.io&ei=3tx9Y8T4DrWnkdUPyq2A8AM&ved=0ahUKEwjE37ib9MP7AhW1U6QEHcoWAD4Q4dUDCA8&uact=5&oq=%22can+be+retried%22+site%3Aprisma.io&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQA0oECEEYAUoECEYYAFCvCFj0OWCXPGgCcAB4AoAB2AGIAY0MkgEGMTIuMy4xmAEAoAEBwAEB&sclient=gws-wiz-serp)